routes/target.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var router = express.Router();
+const db = require("../model/helper");
+
+// Postman Test = OK http://localhost:5000/api/target/test
+router.get("/test", (req, res) => {
+    res.send("Test route is working!");
+  });
+
+// Postman Test = OK http://localhost:5000/api/target/targets
+router.get("/targets", async (req, res) => {
+    console.log('GET /api/target/targets route hit');
+    try {
+      // Search the database to retrieve all the targets
+      const result = await db("SELECT * FROM target");
```

Comment:
As `db` is an instance of a MySQL connection, you should use `db.query` instead. This is typical when using database helpers.

`await db.query("SELECT * FROM targets");`

Reply:
I receive error 500 with the message "Database query failed" when i use await db.query("SELECT * FROM targets"). 
With just await db("SELECT * FROM target"), I receive the status 200 response.